reference/dic_tags.rst

```diff
@@ -14,6 +14,8 @@ Tag Name                                  Usage
 `auto_alias`_                             Define aliases based on the value of container parameters
 `console.command`_                        Add a command
 `container.hot_path`_                     Add to list of always needed services
+`container.no_preload`_                   Remove a class from the list of classes to preload by PHP
```

Comment:
to not preload?

Reply:
I've reworded as "Remove a class from the list of classes preloaded by PHP"